clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2811,10 +2811,13 @@ be applied multiple times to the same loop.
     [[intel::max_concurrency(2)]] for (int i = 0; i != 10; ++i) a[i] = 0;
   }
 
+  [[intel::component_max_concurrency(2)]] void foo1 { }
```

Comment:
You are adding new spelling here. You did not add it inside Attr.td file and also CodeGen test is using loop attribute spelling [[intel::max_concurrency()]]. Are we going to use same loop attribute spelling as function attribute?